keylime/measured_boot.py

```diff
@@ -72,7 +73,7 @@ def evaluate_policy(
     mb_policy: Optional[Policy],
     mb_policy_name: str,
     mb_refstate_data: RefState,
-    mb_measurement_data: str,
+    mb_measurement_data: tests.Data,
```

Comment:
This goes into this here `mb_policy.evaluate(mb_refstate_data, mb_measurement_data)` with signature `def evaluate(self, refstate: RefState, eventlog: tests.Data) -> str:`. A bit odd that our tools didn't detect this...

Reply:
It is because `str` is a strict subset of `tests.Data` so it works.